hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -331,11 +332,11 @@ protected HStore(final HRegion region, final ColumnFamilyDescriptor family,
       confPrintThreshold = 10;
     }
     this.parallelPutCountPrintThreshold = confPrintThreshold;
-    LOG.info("Store={},  memstore type={}, storagePolicy={}, verifyBulkLoads={}, "
-            + "parallelPutCountPrintThreshold={}, encoding={}, compression={}",
+    LOG.info("Created HStore: Store={},  memstore type={}, storagePolicy={}, verifyBulkLoads={}, "
+            + "parallelPutCountPrintThreshold={}, encoding={}, compression={} in region {}",
         getColumnFamilyName(), memstore.getClass().getSimpleName(), policyName, verifyBulkLoads,
         parallelPutCountPrintThreshold, family.getDataBlockEncoding(),
-        family.getCompressionType());
+        family.getCompressionType(), getRegionInfo().getRegionNameAsString());
```

Comment:
Encoded name, not full name.

Reply:
> Encoded name, not full name.

OK

Reply:
Oh ya encoded name..  In fact  I wanted to say , better we can use RegionInfo#prettyPrint(encodedRegionName).. Sorry missed adding that.